packages/@aws-cdk/aws-batch-alpha/lib/ecs-container-definition.ts

```diff
@@ -896,6 +896,13 @@ export interface IEcsFargateContainerDefinition extends IEcsContainerDefinition
    * @default LATEST
    */
   readonly fargatePlatformVersion?: ecs.FargatePlatformVersion;
+
+  /**
+   * The size for ephemeral storage.
+   *
+   * @default - no storage
+   */
+  readonly ephemeralStorage?: Size;
```

Comment:
```suggestion
  readonly ephemeralStorageSize?: Size;
```

just in case they add new properties to the CFN object type